experiments/scripts/subj-scripts/ambari-server.sh

```diff
@@ -12,7 +12,8 @@ export NEW_DT_SUBJ=$NEW_DT_SUBJ_ROOT/ambari-server/target
 # Directory containing the new subject's src directory.
 export NEW_DT_SUBJ_SRC=$NEW_DT_SUBJ_ROOT/ambari-server
 
-# The name of the subject you wish to add (e.g., xml_security) 
+# The name of the subject you wish to add (e.g., xml_security)
 export SUBJ_NAME=ambari-server
-# The name of the subject you want to be displayed in the paper (e.g., XML Security) 
+# The name of the subject you want to be displayed in the paper (e.g., XML Security)
 export SUBJ_NAME_FORMAL=Ambari-Server
+
```

Comment:
Please have this script `source ../setup-vars.sh` so that the user won't have to do one more step to setup the variables.